k8s/kops-weave/genie-plugin.yaml

```diff
@@ -128,6 +128,17 @@ spec:
       hostNetwork: true
       hostPID: true
       serviceAccountName: genie-plugin
+      priorityClassName: system-node-critical
+      initContainers:
```

Comment:
While running the `autoscaler`, I noticed that it is possible for `cnigenie` to start before `flannel` is installed, which then borks the given host.